content/en/tracing/trace_collection/library_config/_index.md

```diff
@@ -16,13 +16,79 @@ To instrument an application written in a language that does not yet have offici
 ## Common configuration options
 The following configuration options behave consistently across the latest versions of all Datadog SDKs, unless otherwise noted:
 
+### Agent
+
+`DD_TRACE_AGENT_URL`
+: **Default**: `http://localhost:8126` <br>
+**Supported Input**: A string representing an HTTP or UDS url <br>
+**Description**: The URL for connecting the tracer to the Datadog agent. Valid URL schemas include `http://` and `unix://` (UNIX Domain Sockets). This value takes precedence over `DD_AGENT_HOST` and `DD_TRACE_AGENT_PORT` if set.
+
+`DD_DOGSTATSD_PORT`
+: **Default**: `8125` <br>
+**Supported Input**: Integer between 0 and 65535 <br>
+**Caveats**: Not supported in C++, Ruby, .NET<br>
+**Description**: Specifies the port number for the DogStatsD server to which runtime metrics are sent.
+
+`DD_DOGSTATSD_HOST`
+: **Default**: `localhost` <br>
+**Supported Input**: String representing a hostname or IP address <br>
+**Caveats**: Not supported in C++, Ruby, .NET<br>
+**Description**: Specifies the hostname or IP address of the DogStatsD server to which runtime metrics are sent.
+
+### Unified Service Tagging
+
+`DD_VERSION`
+: **Default**: `null`<br>
+**Supported Input**: A string representing an application version<br>
+**Caveats**: Node.js defaults to the version number from package.json<br/>
+**Description**: Adds a `version` tag to all spans, except for [Inferred services][3]
+
+`DD_SERVICE`
+: **Default**: `null`, the SDK tries to automatically determine a service name<br>
+**Supported Input**: A string representing an application service name <br>
+**Description**: Sets the default service name used for most spans. SDKs may set a different service name for inferred services. Integration spans may use their own default names, which can differ from the value specified in `DD_SERVICE`
+
+`DD_ENV`
+: **Default**: `null` <br>
+**Supported Input**: A string representing an application environment name (for example, `prod`, `dev`) <br>
+**Description**: Adds an environment tag to all spans generated by the tracer instance.
+
+`DD_TAGS`
+: **Default**: `null` <br>
+**Supported Input**: A string representing key value pairs that are delimited by a colon and separated by a comma and/or a space (for example, `<key1>:<value1>, <key2>:<value2>`>) <br>
+**Description**: Tags to apply to produced data. Must be a list of `<key>:<value>` separated by commas and/or spaces.
+
+### Diagnostics
+
+`DD_TRACE_LOG_DIRECTORY`
+: **Default**: Varies by SDK, environment, and runtime. Please read more in the specific configuration page above for your chosen language <br>
+**Supported Input**: A valid full or relative directory path that exists on the system<br>
+**Caveats**: Not supported in Java, Node.js, Ruby, Python<br>
+**Description**: Specifies the directory where tracer log files should be routed. If the directory does not exist, the SDK falls back to its default diagnostic logging method.
+
+`DD_RUNTIME_METRICS_ENABLED`
```

Comment:
nit: I don't think this belongs in the Diagnostics section. I think would be best classified under Metrics since this generates metrics that get sent to the Datadog platform